### ISSUE TITLE
problem for dataset

### ISSUE THREAD

**Issue by haha99000** [xiehousheng/SemLA#2]
could you please provide another download link to the COCO dataset ? I can't download from Drive.

**Comment by xiehousheng** [comment]
Thank you for taking note of this issue. We have updated the coco dataset link to https://cocodataset.org/#download

**Comment by haha99000** [comment]
Thanks for your reply ! ! ! Can you provide the generate pseudo-infrared images for each image in the COCO and IVS dataset using [CPSTN] ?

**Comment by xiehousheng** [comment]
This data is too large, we apologize that we are unable to provide it directly at this time. Directly generating the corresponding pseudo infrared image using CPSTN on your local device can ensure that it corresponds to the COCO dataset you downloaded.

**Comment by xuliuwei** [comment]
@xiehousheng could you please tell me how the training set and the test set of the experiment are divided?
